projects/parsers/example.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
```

Comment:
@rpwils we typically include the following header. Can you update all of the files?

"""
description: <short description>
version: <>
created: <date>
author: <name>
dependencies:
    * <>
license: BSD-2-Clause
"""

Reply:
I would be open to changing to that style